app/modules/eus/enduser_setup.html

```diff
@@ -242,7 +244,11 @@ <h4 id='st'>Updating Status...</h4>
 		}
 		function submit() {
 			submitted = true;
-			var url = '/setwifi?wifi_ssid=' + encodeURIComponent($('#ssid').value) + '&wifi_password=' + encodeURIComponent($('#wifi_password').value);
+			var json, config = {};
+			config.twitterAPI = $('#TwitterAPI').value;
+			config.twitterMessage = $('#TwitterMessage').value;
+			json = JSON.stringify(config);
+			var url = '/setwifi?wifi_ssid=' + encodeURIComponent($('#ssid').value) + '&wifi_password=' + encodeURIComponent($('#wifi_password').value) + '&config=' + encodeURIComponent(json);
 			clearTimeout(rs);
```

Comment:
Likewise, these shouldn't be part of the default page. It'd be a good example in the docs though.